stream-applications-release-train/pom.xml

```diff
@@ -22,65 +22,7 @@
-        <zeromq-source.version>${apps.version}</zeromq-source.version>
-        <cassandra-sink.version>${apps.version}</cassandra-sink.version>
-        <analytics-sink.version>${apps.version}</analytics-sink.version>
-        <elasticsearch-sink.version>${apps.version}</elasticsearch-sink.version>
-        <file-sink.version>${apps.version}</file-sink.version>
-        <ftp-sink.version>${apps.version}</ftp-sink.version>
-        <geode-sink.version>${apps.version}</geode-sink.version>
-        <jdbc-sink.version>${apps.version}</jdbc-sink.version>
-        <log-sink.version>${apps.version}</log-sink.version>
-        <mongodb-sink.version>${apps.version}</mongodb-sink.version>
-        <mqtt-sink.version>${apps.version}</mqtt-sink.version>
-        <rabbit-sink.version>${apps.version}</rabbit-sink.version>
-        <redis-sink.version>${apps.version}</redis-sink.version>
-        <router-sink.version>${apps.version}</router-sink.version>
-        <rsocket-sink.version>${apps.version}</rsocket-sink.version>
-        <pgcopy-sink.version>${apps.version}</pgcopy-sink.version>
-        <s3-sink.version>${apps.version}</s3-sink.version>
-        <sftp-sink.version>${apps.version}</sftp-sink.version>
-        <tcp-sink.version>${apps.version}</tcp-sink.version>
-        <throughput-sink.version>${apps.version}</throughput-sink.version>
-        <twitter-message-sink.version>${apps.version}</twitter-message-sink.version>
-        <twitter-update-sink.version>${apps.version}</twitter-update-sink.version>
-        <wavefront-sink.version>${apps.version}</wavefront-sink.version>
-        <websocket-sink.version>${apps.version}</websocket-sink.version>
-        <zeromq-sink.version>${apps.version}</zeromq-sink.version>
-        <aggregator-processor.version>${apps.version}</aggregator-processor.version>
-        <bridge-processor.version>${apps.version}</bridge-processor.version>
-        <filter-processor.version>${apps.version}</filter-processor.version>
-        <groovy-processor.version>${apps.version}</groovy-processor.version>
-        <header-enricher-processor.version>${apps.version}</header-enricher-processor.version>
-        <http-request-processor.version>${apps.version}</http-request-processor.version>
-        <image-recognition-processor.version>${apps.version}</image-recognition-processor.version>
-        <semantic-segmentation-processor.version>${apps.version}</semantic-segmentation-processor.version>
-        <object-detection-processor.version>${apps.version}</object-detection-processor.version>
-        <script-processor.version>${apps.version}</script-processor.version>
-        <splitter-processor.version>${apps.version}</splitter-processor.version>
-        <transform-processor.version>${apps.version}</transform-processor.version>
-        <twitter-trend-processor.version>${apps.version}</twitter-trend-processor.version>
+		<apps.docker.tag>${apps.version}</apps.docker.tag>
```

Comment:
I will fix this formatting shortly. All of my changes to xml files are off in spacing.